package.json

```diff
@@ -27,6 +28,7 @@
     "@vitejs/plugin-vue": "^2.3.4",
     "autoprefixer": "^10.4.8",
     "buffer": "^6.0.3",
+    "chroma-js": "^2.4.2",
```

Comment:
It looks like you import Chroma in a few places, but it's not actually used. Do we need it?

Reply:
ah, oops, i probably should have marked this PR as a draft again after i made some dramatic edits. chroma is there to do some colour manipulation for the hover effect on e.g. condition meters that have a thematic colour. i scrapped and my (rather baroque) initial attempt at that, but i'm going to re-implement a simpler version that actually uses the library.